lib/ui/platform_dispatcher.dart

```diff
@@ -487,8 +487,11 @@ class PlatformDispatcher {
   }
 
   late _SetNeedsReportTimingsFunc _setNeedsReportTimings;
-  void _nativeSetNeedsReportTimings(bool value)
-      native 'PlatformConfiguration_setNeedsReportTimings';
+
+  void _nativeSetNeedsReportTimings(bool value) => __nativeSetNeedsReportTimings(value);
```

Comment:
I noticed various APIs like this where a private Dart function like `_nativeSetNeedsReportTimings` delegates to another function with an `@FfiNative` annotation like `__nativeSetNeedsReportTimings`.

Is the second layer of indirection needed?

Reply:
Right. This is a quirk that popped up a few times where Dart instance methods mapped to native static functions.
`_nativeSetNeedsReportTimings` is an instance method, so adding the `@FfiNative` annotation on it would cause `PlatformConfigurationClient::SetNeedsReportTimingsOrThrow` to be called as if it too is an instance method, which of course doesn't work out - hence the indirection.

Here `_nativeSetNeedsReportTimings` doesn't actually need to be an instance method, so the right thing to do would be to refactor it into a static function.
I have however deliberately chosen not to refactor those cases as I didn't want to change more existing API than I had to in this PR.

Would you like me to include that refactoring instead?